converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java

```diff
@@ -1,6 +1,8 @@
 package gov.cms.qpp.conversion.validate;
 
 import com.google.common.base.Strings;
+import com.google.common.collect.Collections2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.collect.Collections2'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)